.github/workflows/build.yml

```diff
@@ -526,6 +526,22 @@ jobs:
       - run: cargo install taplo-cli --locked
       - run: taplo format --check
 
+  typos:
+    name: Typos
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Install rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+      - run: cargo install typos-cli --locked
+      # NOTE: separate step seeems to be needed to check `.*.toml` - TODO NEEDS INVESTIGATION
+      - run: typos .*.toml
```

Comment:
Is this a regexp, or is shell expansion handling this? If it's a regexp, I'd recommend quoting the pattern and escaping the second `.`. If it's shell expansion, maybe still worth putting in `"`'s?

Reply:
resolved in new `admin/typos` script